src/test/fuzz/addrman.cpp

```diff
@@ -114,4 +206,50 @@ FUZZ_TARGET_INIT(addrman, initialize_addrman)
     (void)addr_man.size();
     CDataStream data_stream(SER_NETWORK, PROTOCOL_VERSION);
     data_stream << addr_man;
+
+    // Check that serialize followed by unserialize produces the same addrman.
+
+    if (fuzzed_data_provider.remaining_bytes() < 100) {
+        return;
+    }
+
+    data_stream.clear();
```

Comment:
Is there a reason to not create a new testcase for this?

Reply:
I appended to the existent test so that addrman is even in a more fancy state, left from the `CallOneOf()` tests above this. But I see how they are consuming precious fuzz bytes...